demisto_sdk/commands/format/update_generic.py

```diff
@@ -220,95 +219,58 @@ def regex_matching_key(self, field, schema_keys):
-            if self.verbose:
-                click.echo('Setting fromVersion field')
-            # If current file does not have fromversion key
-            if self.from_version_key not in self.data:
-                # If user entered specific from version key to be set
-                if from_version:
-                    self.data[self.from_version_key] = from_version
-                # if it is new contributed pack = setting version to 6.0.0
-                elif should_set_from_version:
-                    self.data[self.from_version_key] = VERSION_6_0_0
-                # Otherwise add fromversion key to current file and set to default 5.5.0
-                else:
-                    self.data[self.from_version_key] = NEW_FILE_DEFAULT_5_5_0_FROMVERSION
-            # If user wants to modify fromversion key and the key already existed
-            elif from_version:
-                self.data[self.from_version_key] = from_version
-            # if it is new contributed pack, this is integration, and its version is 5.5.0 do not change it
-            # if it is new contributed pack = setting version to 6.0.0
-            elif should_set_from_version:
-                if self.data.get(self.from_version_key) != '5.5.0' or file_type != INTEGRATION:
-                    self.data[self.from_version_key] = VERSION_6_0_0
-
-            # If it is new pack, and it has from version lower than 5.5.0, ask to set it to 5.5.0
-            # Playbook has its own validation in update_fromversion_by_user() function in update_playbook.py
-            elif LooseVersion(self.data.get(self.from_version_key, DEFAULT_CONTENT_ITEM_FROM_VERSION)) < \
-                    LooseVersion(NEW_FILE_DEFAULT_5_5_0_FROMVERSION) and file_type != PLAYBOOK:
-                if self.assume_yes:
-                    self.data[self.from_version_key] = NEW_FILE_DEFAULT_5_5_0_FROMVERSION
-                else:
-                    set_from_version = str(
-                        input(f"\nYour current fromversion is: '{self.data.get(self.from_version_key)}'. Do you want "
-                              f"to set it to '5.5.0'? Y/N ")).lower()
-                    if set_from_version in ['y', 'yes']:
-                        self.data[self.from_version_key] = NEW_FILE_DEFAULT_5_5_0_FROMVERSION
-
-        # If there is an existing file in content repo
+        current_fromversion_value = self.data.get(self.from_version_key)
+        if self.verbose:
+            click.echo('Setting fromVersion field')
+
+        # If user asked specific fromVersion to set.
+        if self.from_version:
+            self.data[self.from_version_key] = self.from_version
+        # If there is fromVersion key in the old file(repo).
+        elif self.old_file.get(self.from_version_key):
+            if not current_fromversion_value:
+                self.data[self.from_version_key] = self.old_file.get(self.from_version_key)
+        # If its new file, ask the user to generate it.
```

Comment:
```suggestion
        if self.from_version:
            self.data[self.from_version_key] = self.from_version
        elif self.old_file.get(self.from_version_key):
            if not current_fromversion_value:
                self.data[self.from_version_key] = self.old_file.get(self.from_version_key)
```
avoid adding comments when the code is clear (which is the case now  🎉 )